src/dotnet/OpenAPI.Validator/Model/Utilities/ValidationUtilities.cs

```diff
@@ -23,6 +23,36 @@ public static class ValidationUtilities
         public static readonly Regex ListBySidRegEx = new Regex(@".+_(List|ListBySubscriptionId|ListBySubscription|ListBySubscriptions)$", RegexOptions.IgnoreCase);
         public static readonly Regex ListByRgRegEx = new Regex(@".+_ListByResourceGroup$", RegexOptions.IgnoreCase);
 
+        public static IEnumerable<string> GetTenantResourceModels(IEnumerable<string> resourceModels, ServiceDefinition serviceDefinition)
+        {
+            List<string> tenantResourceModels = new List<string>();
+            if(serviceDefinition.Paths.Any())
+            {
+                foreach (KeyValuePair<string, Dictionary<string, Operation>> path in serviceDefinition.Paths)
+                {
+                    if (!path.Key.Contains("/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/"))
```

Comment:
do we need a regex for this? The names in between {} could vary based on what the author of the spec decides to call them as they are parameters, and casing may be different too. 
I believe there are cases in which one or the other may no be there too, should those also be excluded, the description from Gaurav in the issue says "will not have the /resourceGroups or /subscriptions section in the URL", so the "or" makes me wonder.